zyte_api/__main__.py

```diff
@@ -71,13 +72,13 @@ def write_output(content):
                 try:
                     result = await fut
                 except Exception as e:
-                    if store_errors:
-                        write_output(e.parsed.response_body.decode())
+                    if store_errors and isinstance(e, RequestError):
+                        write_output(e.parsed.data)
```

Comment:
This was an error detected while addressing typing issues, and removing the request error mock from tests. `write_output` does `json.dumps`, so when passing a string here before, it was printing in the file `'{"…` instead of the expected `{"…`.